packages/design-system/src/components/command.tsx

```diff
@@ -30,23 +30,23 @@ import { Button } from "./button";
 import { Popover, PopoverContent, PopoverTrigger } from "./popover";
 import { Kbd } from "./kbd";
 
-const panelWidth = "400px";
-const itemHeight = "32px";
+const panelWidth = 400;
+const itemHeight = 32;
 const inputBorderBottomSize = "--command-input-border-bottom-width";
 
 const StyledCommand = styled(CommandPrimitive, {
   boxSizing: "border-box",
   width: panelWidth,
-  borderRadius: theme.borderRadius[4],
   boxShadow: theme.shadows.menuDropShadow,
   backgroundColor: theme.colors.backgroundControls,
   border: `1px solid ${theme.colors.borderMain}`,
+  borderRadius: theme.borderRadius[7],
   // clip selected item background
   overflow: "clip",
   // remove input border bottom when no command matches
-  [inputBorderBottomSize]: "0px",
+  [inputBorderBottomSize]: 0,
   "&:has([cmdk-group]:not([hidden]))": {
-    [inputBorderBottomSize]: "1px",
+    [inputBorderBottomSize]: 1,
```

Comment:
You sure `1 solid red` will work?

Reply:
no, but we compile it to 1px, so we can use numbers, right?

Reply:
who compiles it? Stitches magic? I have no idea if it compiles custom properties or not. Too magic.